versions/0_5/flake.nix

```diff
@@ -0,0 +1,30 @@
+{
+  inputs =
+    {
+      holochain = {
+        url = "github:holochain/holochain/holochain-0.5.0-dev.21";
+        flake = false;
+      };
+
+      lair = {
+        url = "github:holochain/lair/lair_keystore-v0.5.3";
+        flake = false;
+      };
+
+      # holochain_cli_launch
+      launcher = {
+        url = "github:holochain/hc-launch/holochain-weekly";
+        flake = false;
+      };
+
+      # holochain_scaffolding_cli
+      scaffolding = {
+        url = "github:holochain/scaffolding/holochain-weekly";
+        flake = false;
+      };
+    };
+
+  outputs = { ... }: {
+    rustVersion = "1.85.0";
```

Comment:
The `develop-0.5` branch will have to be updated separately to use the `0_5` version and update its workspace Rust version to 1.85